src/ckb/network.rs

```diff
@@ -66,6 +66,11 @@ pub struct AcceptChannelResponse {
     pub new_channel_id: Hash256,
 }
 
+#[derive(Copy, Clone, Debug, Serialize)]
+pub enum ControlChannelResponse {
```

Comment:
I think your intention here is to encompass all the responses to`ControlPcnChannel` messages to this enum? In my opinion, return a concrete type like `AcceptChannelResponse` is better here, i.e. instead of

```
    AddTlc(
        AddTlcCommand,
        #[serde(skip)] Option<RpcReplyPort<Result<ControlChannelResponse, RpcError>>>,
    ),
```

we can define

```
    AddTlc(
        AddTlcCommand,
        #[serde(skip)] Option<RpcReplyPort<Result<AddTlcResponse, RpcError>>>,
    ),
```

because the client can always expect a response to `AddTlc` to be a concrete response type which contains the tlc id. So making a enum for all responses seems to have no real benefits. If we are to use the same enum for all responses, then the client has to pattern match results for a specific response type. This seems strange to me.

Reply:
Yes, my first version is encompassing all the responses to `ControlPcnChannel`, then I changed direction and forgot on this.